pandas/tests/extension/json/test_json.py

```diff
@@ -240,6 +240,10 @@ class TestReduce(base.BaseNoReduceTests):
 
 
 class TestMethods(BaseJSON, base.BaseMethodsTests):
+    @pytest.mark.xfail(reason="ValueError: setting an array element with a sequence")
+    def test_hash_pandas_object(self, data):
+        super().test_hash_pandas_object(data)
```

Comment:
could you expand on this please? why are we adding it, why does it fail, is it expected that it should pass?